streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -796,3 +815,20 @@ package object testPackage extends Assertions {
     }
   }
 }
+
+/**
+ * Helper methods for testing StreamingContextSuite.
+ * This includes methods to access private methods and fields in ExecutorAllocationManager.
```

Comment:
Apologies for missing that. Added in the latest commit with the sources variable and the changing of the comment text.